src/pages/index.js

```diff
@@ -150,56 +112,49 @@ function Home() {
         {features && features.length > 0 && (
           <section className={styles.features}>
             <div className="container">
-              <div className="row">
+              <div className={styles.featuresGrid}>
                 {features.map((props, idx) => (
                   <Feature key={idx} {...props} />
                 ))}
               </div>
             </div>
           </section>
         )}
-        {features2 && features2.length > 0 && (
-          <section className={styles.features2}>
-            <div className="container">
-              <div className="row">
-                {features2.map((props, idx) => (
-                  <Feature2 key={idx} {...props} />
-                ))}
-              </div>
-            </div>
-          </section>
-        )}
         <div className={clsx("hero hero--secondary", styles.heroBanner)}>
           <div className="container">
-            <h2 className="hero__title">Unsure where to begin?</h2>
-            <p className="hero__subtitle">Try one of our use case guides</p>
+            <h2 className="hero__title">Want to know what's new?</h2>
+            <p className="hero__subtitle">
+              Check out some of our latest features
+            </p>
             <div className={styles.buttons}>
               <Link
                 className={clsx(
                   "button button--outline button--secondary button--lg button--hero get-started-use-case",
                   styles.getStarted
                 )}
-                to={useBaseUrl("docs/guides/orchestrate-microservices/")}
+                to={useBaseUrl(
+                  "docs/apis-tools/spring-zeebe-sdk/getting-started.md"
+                )}
               >
-                Microservice Orchestration
+                Spring Zeebe SDK
               </Link>
               <Link
                 className={clsx(
                   "button button--outline button--secondary button--lg button--hero get-started-use-case get-started-use-case-2",
                   styles.getStarted
                 )}
-                to={useBaseUrl("docs/guides/orchestrate-human-tasks/")}
+                to={useBaseUrl("docs/reference/release-notes/850.md")}
```

Comment:
@christinaausley I did leave a [landing page](https://docs.camunda.io/docs/next/reference/release-notes/) for release notes. This would save a manual step on release day and (hopefully) make it easier for people to find alpha release notes as well. Do you still prefer to link directly to the latest minor release notes?

Reply:
No I love this!